sdk/keyvault/keyvault-keys/README.md

```diff
@@ -257,6 +257,10 @@ When you submit a pull request, a CLA-bot will automatically determine whether y
 a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions
 provided by the bot. You will only need to do this once across all repos using our CLA.
 
+## Developer Contributing Guide
+
+For details on developer inner-loop workflow and contributing guide see the [contributing guide](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md)
```

Comment:
Do relative links work? If so I'd suggest we do that instead of hard-coding the master branch.

Reply:
Yes just tested the relative links in a draft PR - they seem to work https://github.com/Azure/azure-sdk-for-js/pull/4274/files#diff-e18032b93fceab824beef069b8677835

Reply:
OK lets use those as they allow people to stay scoped inside the branch they are in which will help when looking at older branches overtime.